.env

```diff
@@ -25,9 +25,4 @@ FAVICON_URL=''
 COLLECT_YEAR_OF_BIRTH=true
 APP_ID=''
 MFE_CONFIG_API_URL=''
-ENABLE_SKILLS_BUILDER=''
 ENABLE_SKILLS_BUILDER_PROFILE=''
```

Comment:
Can you give context on why we would keep this flag?

Reply:
That is the flag used by the Profile page to decide whether or not to render the learner's goal. I think we still want this flag unless there are plans to remove the feature altogether